package.json

```
@@ -1,54 +1,42 @@
 {
+  "type": "RESTApp",
   "name": "restapp-boilerplate",
-  "version": "2.0.1",
+  "version": "1.0.0",
   "description": "A RESTApp boilerplate",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/strt/restapp-boilerplate.git"
-  },
-  "scripts": {
-    "setup": "node util/setup.js",
-    "zip": "npm run transpile && node util/zip.js",
-    "deploy": "npm run zip && node util/deploy.js",
-    "force-deploy": "npm run zip && node util/deploy.js force",
-    "create-addon": "node util/create-addon.js",
-    "sign": "npm run zip && node util/sign.js",
-    "prod-deploy": "node util/prod-deploy.js",
-    "transpile": "npx babel ./src -d ./compiled --copy-files --delete-dir-on-start",
-    "precommit": "lint-staged"
-  },
   "license": "MIT",
+  "repository": "github:strt/restapp-boilerplate.git",
+  "homepage": "https://github.com/strt/restapp-boilerplate#readme",
+  "bugs": {
+    "url": "https://github.com/strt/restapp-boilerplate/issues"
+  },
   "author": "Strateg Marknadsföring AB",
   "contributors": [
     "Alexander Karlsson <alexander.karlsson@strateg.se>",
     "Alexander Nanberg <alexander.nanberg@strateg.se>",
     "Jesper Johansson <jesper.johansson@strateg.se>"
   ],
-  "bugs": {
-    "url": "https://github.com/strt/restapp-boilerplate/issues"
+  "scripts": {
+    "build": "sitevision-scripts build",
+    "setup": "sitevision-scripts setup",
+    "deploy": "sitevision-scripts deploy",
+    "create-addon": "sitevision-scripts create-addon",
+    "sign": "sitevision-scripts sign",
+    "prod-deploy": "echo 'todo'"
+  },
+  "lint-staged": {
+    "src/**/*.js": [
+      "eslint --fix",
+      "prettier-eslint --write",
+      "git add"
+    ]
   },
-  "homepage": "https://github.com/strt/restapp-boilerplate#readme",
   "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-transform-object-assign": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
     "eslint": "^5.5.0",
     "eslint-config-strateg": "^1.0.1",
     "husky": "^0.14.3",
-    "inquirer": "^5.2.0",
-    "jsonfile": "^4.0.0",
     "lint-staged": "^7.0.5",
     "prettier": "^1.14.2",
     "prettier-eslint-cli": "^4.7.1",
-    "request": "^2.87.0",
-    "zip-dir": "^1.0.2"
-  },
-  "lint-staged": {
-    "src/**/*.js": [
-      "eslint --config '.eslintrc' --fix",
-      "git add"
-    ]
+    "sitevision-scripts": "^0.1.0"
```

```suggestion
    "sitevision-scripts": "^0.3.0"
```

Bump to most recent version.